db/migrate/20150724143942_set_completed_at_field.rb

```diff
@@ -0,0 +1,11 @@
+class SetCompletedAtField < ActiveRecord::Migration
+  def up
+    Item.completed.each do |item|
+      item.update_attributes(completed_at: item.updated_at)
+    end
```

Comment:
@vlasiak, I suggest to use `update_all('completed_at = updated_at')`, does it make a sense?